book/special.qmd

```diff
@@ -851,188 +851,188 @@ The experiment shows that using the K-means algorithm with four centers has the
+% Below we only show classification examples but regression follows trivially.
+%
+% ```{r special-044, message=FALSE, warning=FALSE}
+% library(mlr3spatial)
+% library(mlr3spatiotempcv)
+%
+% # create task from `data.frame`
+% task = as_task_classif_st(ecuador, id = "ecuador_task",
+%   target = "slides", positive = "TRUE",
+%   coordinate_names = c("x", "y"), crs = 32717)
+%
+% # or create task from 'sf' object
+% data_sf = sf::st_as_sf(ecuador, coords = c("x", "y"), crs = 32717)
+% task = as_task_classif_st(data_sf, target = "slides", positive = "TRUE")
+% task
+% ```
+%
+% Once a task is created, you can train and predict as normal.
+%
+% ```{r special-045}
+% lrn("classif.rpart")$train(task)$predict(task)
+% ```
+%
+% However as discussed above, it is best to use the specialized resampling methods in order to have trustworthy estimates of model performance.
+%
+% ### Spatiotemporal Cross-Validation {#spatiotemp-cv}
+%
+% Before we look at the spatial resampling methods implemented in `r mlr3spatiotempcv` we will first show what can go wrong if standard resampling methods are used.
+% Below we benchmark a decision tree on the `r ref("mlr_tasks_ecuador")` task using two different repeated cross-validation resampling methods, the first ("NSpCV") is a standard resampling method from `r mlr3`, the second ("SpCV") is from `r mlr3spatiotempcv` and is optimized for spatial data.
+% The example highlights how "NSpCV" makes it appear as if the decision tree is performing better than it is with significantly higher estimated performance, however this is an overconfident prediction due to the autocorrelation in the data.
+%
+% ```{r special-046, warning=FALSE,message=FALSE}
+% learner = lrn("classif.rpart", predict_type = "prob")
+% rsmp_nsp = rsmp("repeated_cv", folds = 3, repeats = 2, id = "NSpCV")
+% rsmp_sp = rsmp("repeated_spcv_coords", folds = 3, repeats = 2, id = "SpCV")
+%
+% design = benchmark_grid(task, learner, c(rsmp_nsp, rsmp_sp))
+% bmr = benchmark(design)
+% bmr$aggregate(msr("classif.acc"))[, c(5, 7)]
+% ```
+%
+% In the above example, the non-spatial resampling method means that the train and test data are very similar due to spatial autocorrelation, and so this is almost as bad as evaluating a model on the training data (see @sec-basics).
+% In contrast, the spatial method has accommodated for autocorrelation and the test data is therefore less correlated (though still some association will remain) with the training data.
+% Visually this can be seen using built-in `r ref("autoplot")` methods.
+% In @fig-sprsmp we visualize how the task is partitioned according to the spatial resampling method (@fig-sprsmp top) and non-spatial resampling method (@fig-sprsmp bottom).
+% There is a clear separation between coordinates using the 'correct' spatial resampling method whereas the train and test splits overlap significantly (and are therefore highly correlated) using the non-spatial method.
```

Comment:
```suggestion
% There is a clear separation in space for the respective partitions when using the spatial resampling whereas the train and test splits overlap a lot (and are therefore more correlated) using the non-spatial method.
```

Given that "significantly" is often associated with a statical meaning I'd suggest to avoid the term in such contexts.

The degree of correlation depends and is not always "high".

"seperation between coordinates" is uncommon wording here. One would rather say "Separation in space for the respective partitions".